sdk/cognitivelanguage/azure-ai-language-conversations/tests/async/test_conversation_app_async.py

```diff
@@ -14,76 +14,56 @@
 
     @GlobalConversationAccountPreparer()
-    async def test_conversation_app(self, conv_account, conv_key, conv_project):
+    async def test_conversation_app(self, endpoint, key, conv_project_name, conv_deployment_name):
 
-        # prepare data
-        query = "One california maki please."
-        input = ConversationAnalysisOptions(
-            query=query,
-        )
-
-        # analyze quey
-        client = ConversationAnalysisClient(conv_account, AzureKeyCredential(conv_key))
-        async with client:
-            result = await client.analyze_conversations(
-                input,
-                project_name=conv_project,
-                deployment_name='production'
-            )
-        
-        # assert
-        assert isinstance(result, AnalyzeConversationResult)
-        assert result.query == query
-        assert isinstance(result.prediction, ConversationPrediction)
-        assert result.prediction.project_kind == 'conversation'
-        assert result.prediction.top_intent == 'Order'
-        assert len(result.prediction.entities) > 0
-        assert len(result.prediction.intents) > 0
-        assert result.prediction.intents[0].category == 'Order'
-        assert result.prediction.intents[0].confidence_score > 0
-        assert result.prediction.entities[0].category == 'OrderItem'
-        assert result.prediction.entities[0].text == 'california maki'
-        assert result.prediction.entities[0].confidence_score > 0
-
-    @GlobalConversationAccountPreparer()
-    async def test_conversation_app_with_dictparams(self, conv_account, conv_key, conv_project):
-        
-        # prepare data
-        query = "One california maki please."
-        params = {
-            "query": query,
-        }
-
-        # analyze quey
-        client = ConversationAnalysisClient(conv_account, AzureKeyCredential(conv_key))
+        # analyze query
+        client = ConversationAnalysisClient(endpoint, AzureKeyCredential(key))
         async with client:
-            result = await client.analyze_conversations(
-                params,
-                project_name=conv_project,
-                deployment_name='production'
+            query = "Send an email to Carol about the tomorrow's demo"
+            result = await client.analyze_conversation(
+                body={
+                    "kind": "CustomConversation",
+                    "analysisInput": {
+                        "conversationItem": {
+                            "participantId": "1",
+                            "id": "1",
+                            "modality": "text",
+                            "language": "en",
+                            "text": query
+                        },
+                        "isLoggingEnabled": False
+                    },
+                    "parameters": {
+                        "projectName": conv_project_name,
+                        "deploymentName": conv_deployment_name,
```

Comment:
This pattern for moving project/deployment into the payload doesn't really match our other cognitive SDKs where it's in the method signature. Has .NET followed this design?
I guess this means they're no longer in the URL path?

Reply:
it's an update to the march swagger
as i mentioned, they're trying to unify CLU with TA apis
so this is why they have things like Task Kind: "CustomConversation", and other parameters like "project name" and "deployment name"